semantic-codeql/src/Language/CodeQL/Tags.hs

```diff
@@ -100,19 +100,22 @@ instance ToTags CodeQL.DatatypeBranch where
     , name = CodeQL.ClassName { text }
     } = yieldTag text Class loc byteRange >> gtags t
 
-instance ToTags CodeQL.HigherOrderTerm where
-  tags t@CodeQL.HigherOrderTerm
-    { ann = loc@Loc { byteRange }
-    , name = CodeQL.LiteralId { text }
-    } = yieldTag text Call loc byteRange >> gtags t
-
 instance ToTags CodeQL.ClasslessPredicateCall where
   tags CodeQL.ClasslessPredicateCall
     { extraChildren
     } = for_ extraChildren $ \x -> case x of
           Prj t@CodeQL.AritylessPredicateExpr {} -> tags t
           _                                      -> pure ()
 
+instance ToTags CodeQL.QualifiedRhs where
+  tags t@CodeQL.QualifiedRhs
+    { ann = loc@Loc { byteRange }
+    , name = expr
+    } = case expr of
+          Just (Prj CodeQL.PredicateName { text }) -> yieldTag text Call loc byteRange >> gtags t
+          _                                          -> gtags t
+
+instance ToTags CodeQL.HigherOrderTerm
```

Comment:
Put it back alphabetically? Just wondering how hard these are going to be to keep track of if/when we add/remove syntax terms.

Reply:
I 🔤 these instances in https://github.com/github/semantic/pull/528/commits/439e96feea6f6fc3d826f9944c5e0e4036027b86.